src/components/widget/input/cdvy-input.directive.js

```diff
@@ -120,7 +120,6 @@ class CodenvyInput {
       }
       // add ng-change attr if it exist in the parent element
       if ('ngChange' === key) {
-        inputElement.attr('data-ng-change', 'isChanged()');
```

Comment:
We cannot use "ng-change" and "ng-model" properly because ng-model add delay and we can see wrong value.